lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
Isn't squashing going to cause anyone with a devstack/sandbox or anyone close following to have serious migration shenanigans?  This isn't limited to us in production.

Does django let you add courseid with '' as the default and in the next migration drop it?

Two larger suggestions from the team (including @fredsmith who'd be doing these shenanigans) are to either drop and recreate the table, or pivot the name of the model.  But I suggest looking at the extra migration first.